scrapy/utils/testproc.py

```diff
@@ -1,13 +1,15 @@
+from __future__ import absolute_import
```

Comment:
I went back and forth when writing FeedExporter tests; first tried to subclass tests from test_commands, then tried to use utilities from scrapy.utils.testproc, but in the end settled on another solution. That's why https://github.com/scrapy/scrapy/commit/973c31f73595c73fac9b825ce7587d93d7055546 happened.

`from __future__ import absolute_import` is always good to have, I'd add it to all Scrapy modules :) It makes imports compatible in Python 2.x and Python 3.x and prevents stupid errors when a file in the current folder shadows stdlib module or other module in sys.path.